### DISCUSSION TITLE
Should I increment the revision for small changes in a pull request?

### DISCUSSION THREAD

**Discussion by mjyut** [void-linux/void-packages#49669] [Q&A]
A few days ago, I created [a pull request](https://github.com/void-linux/void-packages/pull/49578/files) to make small fix to a package. (This has been merged. Thank you.)

However, now that I've thought about it again, I'm now questioning whether it was correct to increase the package revision to fix a small issue. What should I do if I want to make a similar small pull request in the future?

Options I'm considering:

* Increase revision
* Do not change revision
* Do a different approach (such as setting revision to `0` ?)

**Comment by icp1994** [comment]
If a patch changes files for a package, either source or built, it needs to increment revision. `revision` is set to `1` when a patch updates package version from upstream. For other patches it remains unaltered.
